packages/vaadin-date-picker/theme/lumo/vaadin-date-picker-styles.js

```diff
@@ -20,6 +20,17 @@ const datePicker = css`
       height: 70vh;
     }
   }
+
+  :host([dir='rtl']) [part='input-field'] ::slotted(input) {
+    /* TODO: fix this in input-container, update all screenshots */
+    -webkit-mask-image: var(--_lumo-text-field-overflow-mask-image);
```

Comment:
It turns out `mask-image` still needs a prefix in latest Chrome 🙈  This explains why I had to update screenshots for new field components as the difference was really tricky to spot.

Added this to preserve correct screenshots for the date-picker and avoid updating them in this PR. 
I will restore prefixed property in `vaadin-input-container` in other PR, and add RTL visual tests.